src/git_changelog/cli.py

```diff
@@ -358,7 +358,6 @@ def build_and_render(
         with open(output, "w") as changelog_file:  # type: ignore[arg-type]
             changelog_file.write("\n".join(lines).rstrip("\n") + "\n")
 
-    # overwrite output file
```

Comment:
Why removing this comment?

Reply:
Hmm, that was not intentional. Probably it happened merging your updates. I'll put it back.